src/h3lib/include/h3api.h

```diff
@@ -43,6 +43,9 @@
 /* For size_t */
 #include <stdlib.h>
 
+/* For constants used throughout H3 */
+#include "constants.h"
```

Comment:
Why is this include added?

Reply:
Many parts of the codebase depended on the constants being defined that they were getting indirectly through inclusion of `H3IndexFat.h`. It was easier to add it in here than in so many places across the codebase, and in some ways it makes sense -- if you can't `#include "h3api.h"` and have enough defined to be able to actually compile the requisite methods, then what kind of API is that?

Reply:
I think `h3api.h` exists for consumers of the library, not for the implementation, and consumers would not need these internal constants. I would prefer to replace `#include "h3IndexFat.h"` with `#include "constants.h"` if that's needed.

Reply:
Will do.

Reply:
Done.